src/atoms/LinkBarLink.js

```diff
@@ -21,7 +21,7 @@ const LinkBarLink = ({
 
 LinkBarLink.propTypes = {
 	isActive: propTypes.bool,
-	linkText: propTypes.string,
```

Comment:
Not able to comment on lines that are not changes, But @misund could you explain line 15 to me?